pkg/aggregator/aggregator.go

```diff
@@ -36,6 +37,30 @@ const bucketSize = 10                         // fixed for now
 // MetricSamplePoolBatchSize is the batch size of the metric sample pool.
 const MetricSamplePoolBatchSize = 32
 
+// tagsetSizeThresholdsCount is the size of tagsetSizeThresholds
```

Comment:
It's way more than that, isn't it? Maybe something more generic like "tagsetSizeThresholdsCount is how many thresholds we want telemetry for."?